specs/progress.md

```diff
@@ -0,0 +1,142 @@
+# fast-progress
+
+## Overview
+
+*Progress* is used to display the length of a process or to represent an unspecified wait time. *Progress* can either be circular or linear. *Progress* can either be in an **indeterminate** or **determinate** state. This is determined by whether `value` attribute is passed to the component. 
+
+### Use Cases
+
+The *progress* component should be used for an element that displays the progress status for a task that takes a long time or consists of several steps. Typical use cases include loading elements or status of a download. 
+  
+### Features
+
+A progress should allow the following properties:
+- `value`, the value of the progress, if not passed the progress will be put into its "indeterminate" state.
+- `min-value`, the minimum value.
+- `max-value`, the maximum value.
+- `circular`, whether the progress renders circular or not.
```

Comment:
I would think the styles change but there would be no need for a `circular` attribute.

Reply:
I resolved this in Code but leaving the comment open for discussion, But I believe was should keep it based on @EisenbergEffect comments.

Reply:
I agree with Jane, we should not embed specific style concepts in base component code if we can avoid it.

Reply:
Circular is not a design language specific concept it is a control in its own. You simply can't restyle a linear progress to be circular, it requires different html elements and structure. Unless we are saying we don't won't a circular progress, we will need to write the template in a way that can support it.

Reply:
Can we do this with template changes only though?  I"m not yet clear on how we'd manage/pack that in our system (ie. the single component vs. multiple component).